src/Akeneo/Connectivity/Connection/front/src/connect/components/Marketplace.tsx

```diff
@@ -48,24 +53,51 @@ export const Marketplace: FC<Props> = ({extensions, apps, testApps}) => {
     const security = useSecurity();
     const isManageAppsAuthorized = security.isGranted('akeneo_connectivity_connection_manage_apps');
     const isLimitReached = useConnectionsLimitReached();
-    const extensionsList = extensions.extensions.map((extension: Extension) => (
-        <MarketplaceCard key={extension.id} item={extension} />
-    ));
-    const appsList = apps.apps.map((app: App) => (
-        <MarketplaceCard
-            key={app.id}
-            item={app}
-            additionalActions={[
-                <ActivateAppButton
-                    key={1}
-                    id={app.id}
-                    isConnected={app.connected}
-                    isPending={app.isPending}
-                    isDisabled={!isManageAppsAuthorized || isLimitReached}
-                />,
-            ]}
-        />
-    ));
+    const [search, setSearch] = useState<string>('');
+
+    const extensionsComponents: {[key: string]: JSX.Element} = useMemo(
+        () =>
+            extensions.extensions.reduce((accumulator, extension) => {
+                return {...accumulator, [extension.id]: <MarketplaceCard key={extension.id} item={extension} />};
+            }, {}),
+        [extensions]
+    );
+
+    const extensionsList = extensions.extensions
+        .filter(extension => '' === search || extension.name.toLowerCase().includes(search))
```

Comment:
```suggestion
        .filter(extension => '' === search || extension.name.toLowerCase().includes(search))
```

You transform to lower case the extension name, but not the `search`.
If I'm searching using upper case, like `DEMO APP`, does it work ?

Reply:
fixed, good catch !